src/engine/source/base/CMakeLists.txt

```diff
@@ -41,17 +41,18 @@ endif()
 ####################################################################################################
 # logging
 ####################################################################################################
+# TODO: spdlog includes fmt, but its headers cannot be included as, for example, "fmt/xxx.h" but "fmt/bundled/xxx.h"
 CPMAddPackage(
     NAME fmt
     GITHUB_REPOSITORY fmtlib/fmt
     GIT_TAG "8.1.1"
     )
 
 CPMAddPackage(
-    NAME fmtlog
-    GITHUB_REPOSITORY MengRao/fmtlog
-    GIT_TAG "v2.1.3"
-    DOWNLOAD_ONLY YES
+    NAME spdlog
+    GITHUB_REPOSITORY gabime/spdlog
+    GIT_TAG "v1.11.0"
+    OPTIONS "SPDLOG_FMT_EXTERNAL_HO ON"
     )
```

Comment:
Fix syntax and If use `SPDLOG_FMT_EXTERNAL_HO` then, should be explicit the ftm HO option:
```
CPMAddPackage(
    NAME fmt
    GITHUB_REPOSITORY fmtlib/fmt
    GIT_TAG "8.1.1"
    OPTIONS "FMT_HEADER_ONLY ON"
)

CPMAddPackage(
    NAME spdlog
    GITHUB_REPOSITORY gabime/spdlog
    GIT_TAG "v1.11.0"
    OPTIONS "SPDLOG_FMT_EXTERNAL_HO ON"
)
```

Reply:
Done